dicom/config_service_repositories.go

```diff
@@ -12,6 +12,7 @@ type Repository struct {
 	ID                  string                  `json:"id,omitempty"`
 	OrganizationID      string                  `json:"organizationId"`
 	ActiveObjectStoreID string                  `json:"activeObjectStoreId"`
+	StoreAsComposite    bool                    `json:"storeAsComposite,omitempty"`
```

Comment:
This should be a pointer type to also cover the case where the value is completely missing (as it was before this PR)